llama_index/indices/postprocessor/relevancy/base.py

```diff
@@ -0,0 +1,214 @@
+from llama_index.indices.postprocessor.types import BaseNodePostprocessor
+from enum import Enum
+
+"""
+post_processor_config: Dict = {
```

Comment:
would highly prefer to not have this way of defining a config vs. typed arguments/structs, this UX is not intuitive for users